config/mbed/CMakeLists.txt

```diff
@@ -121,6 +121,25 @@ list(APPEND CHIP_CFLAGS_C ${CMAKE_C_FLAGS_INIT})
 mbed_get_lang_compile_flags(CHIP_CFLAGS_CC CXX)
 list(APPEND CHIP_CFLAGS_CC ${CMAKE_CXX_FLAGS_INIT})
 
+if (CONFIG_CHIP_WITH_EXTERNAL_MBEDTLS)
+    list(APPEND CHIP_CFLAGS 
+        "\"-I${MBED_PATH}connectivity/mbedtls\""
```

Comment:
Would it be possible to propagate all defines and include paths from Mbed to GN ? 
Or even better, extract them from the `mbedtls` [target](https://github.com/ARMmbed/mbed-os/blob/3174a4c6a55c769f51a8fc6185a6fb4fd181d557/connectivity/mbedtls/CMakeLists.txt#L4). It should be possible for includes with [`get_target_property`](https://cmake.org/cmake/help/latest/command/get_target_property.html). I'm don't think it is applicable for defines.

Reply:
I don.t think so. I check the mbed-mbedtls target but it can contain relative paths. Defines are set in headers files
therefore it will not be possible (not directly from the target)

Reply:
Okay, we can get include paths from mbed-mbedtls target